Core.DomainModel/Organization/Organization.cs

```diff
@@ -295,29 +273,55 @@ public Maybe<UIModuleCustomization> GetUiModuleCustomization(string module)
         {
             if (root == null) throw new ArgumentNullException(nameof(root));
 
-            IExternalOrganizationalHierarchyUpdateStrategy strategy;
-            //Pre-validate
+            return GetStsOrganizationConnection(origin)
+                .Bind
+                (
+                    connection =>
+                    {
+                        var strategy = connection.GetUpdateStrategy();
+
+                        var childLevelsToInclude =
+                            levelsIncluded.Select(levels => levels - 1); //subtract the root level before copying
+                        var filteredTree = root.Copy(childLevelsToInclude);
+                        StsOrganizationConnection.SynchronizationDepth =
```

Comment:
here the specific field is also being called even if the param is used in the beginning..